package.json

```diff
@@ -68,6 +68,7 @@
     "yaml-language-server": "^0.7.2"
   },
   "devDependencies": {
+    "@eclipse-che/api": "^7.5.0-SNAPSHOT",
```

Comment:
It's strange to see used pretty old snapshot version. Don't we release that library with the 3 weeks cadence as Che? Or it's the latest one?

Reply:
I see that it's the latest which is surprising for me. Let's try to handle it separately from this PR

Reply:
I got the latest  published version (https://www.npmjs.com/package/@eclipse-che/api).